ngraph/test/pattern.cpp

```diff
@@ -576,7 +577,7 @@ TEST(pattern, recurrent_pattern)
     std::set<std::shared_ptr<pattern::op::Label>> empty_correlated_matches;
     RecurrentMatcher rm(padd, rpattern, empty_correlated_matches);
     ASSERT_TRUE(rm.match(add3));
-    ASSERT_EQ(rm.get_number_of_bound_labels(), 1);
+    ASSERT_EQ(rm.get_number_of_bound_labels(), 3);
```

Comment:
why did you change the number of bound labels? How does it work?

Reply:
Now we start capture local nodes. So that's an expected behavior.